<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vpTrackAttributesPanel</class>
 <widget class="QWidget" name="vpTrackAttributesPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="attributeTree">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="stickyAttributes">
       <property name="toolTip">
        <string>State of attribute check boxes don't change unless by user</string>
       </property>
       <property name="text">
        <string>Sticky</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="displayActiveAttributes">
       <property name="text">
        <string>Active Only</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
